Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md

```diff
@@ -146,20 +147,16 @@ Running  wuauserv           Windows Update
 
 So the service is running, let stop it by typing the following
 
-```powershell-console
-PS C:\> Stop-Service wuauserv
-```
-
-To see the newly created events, type
-
-```powershell-console
-PS C:\> $EventVariable=Get-Event
+```powershell
+PS> Stop-Service wuauserv
 ```
 
+To see the newly created events, type `Get-Event`
 Look at the **MessageData**, it's the same message used in the Register-CimIndicationEvent

```

Comment:
Look at the **MessageData**, it's the same message used in the Register-CimIndicationEvent.